instrumentation/action_view/lib/opentelemetry/instrumentation/action_view/instrumentation.rb

```diff
@@ -7,7 +7,42 @@
 module OpenTelemetry
   module Instrumentation
     module ActionView
-      # The Instrumentation class contains logic to detect and install the ActionView instrumentation
+      # The {OpenTelemetry::Instrumentation::ActionView::Instrumentation} class contains logic to detect and install the ActionView instrumentation
+      #
+      # Installation and configuration of this instrumentation is done within the
+      # {https://www.rubydoc.info/gems/opentelemetry-sdk/OpenTelemetry/SDK#configure-instance_method OpenTelemetry::SDK#configure}
+      # block, calling {https://www.rubydoc.info/gems/opentelemetry-sdk/OpenTelemetry%2FSDK%2FConfigurator:use use()}
+      # or {https://www.rubydoc.info/gems/opentelemetry-sdk/OpenTelemetry%2FSDK%2FConfigurator:use_all use_all()}.
+      #
+      # ## Configuration keys and options
+      #
+      # ### `:disallowed_notification_payload_keys`
+      #
+      # Specifies an array of keys that should be excluded from the notification payload as span attributes.
+      #
+      # ### `:notification_payload_transform`
+      #
+      # - `proc` **default** `nil`
+      #
+      # Specifies custom proc used to extract span attributes form the notification payload.
+      # Use this to rename keys, extract nested values, or perform any other custom logic.
+      #
+      # ### `:legacy_span_names`
+      #
+      # - `boolean`  **default** `false`
+      #
+      # Specifies whether spans names should use the legacy format where the subscription was reverse ordered and white space separated.
+      # If set to `true`, the span name will match the name of the notification itself.
```

Comment:
What do you think about adding examples? It doesn't have to be like this, but as an example:
```suggestion
      # Specifies whether spans names should use the legacy format where the subscription was reverse ordered and white space separated. (Ex. `action_view render_template`)
      # If set to `true`, the span name will match the name of the notification itself. (Ex. `render_template.action_view`)
```